i18n/en-US.yml

```diff
@@ -26,18 +26,238 @@ _name: English
+    tripIsSnoozed: "Unsnooze trip analysis"
+    untogglePause: Resume
+  BeforeSignInScreen:
+    mainTitle: Signing you in
+    message: "In order to access this page you will need to sign in.
+      Please wait while we redirect you to the login page..."
   DefaultItinerary:
     clickDetails: Click to view details
-    # Use ordered placeholders for the departure-arrival string
-    # (this will accommodate right-to-left languages by swapping the order in this string).
-    departureArrivalTimes: "{startTime}—{endTime}"
     # Use ordered placeholders when multiple modes are involved
     # (this will accommodate right-to-left languages by swapping the order/separator in this string).
     multiModeSummary: "{accessMode} to {transitMode}"
-    # If trip is less than one hour only display the minutes.
-    tripDurationFormatZeroHours: "{minutes, number} min"
-    # TODO: Distinguish between one hour (singular) and 2 hours or more?
-    tripDurationFormat: "{hours, number} hr {minutes, number} min"
+  DeleteUser:
+    deleteMyAccount: Delete my account
+  ExistingAccountDisplay:
+    mainTitle: My settings
+    notifications: Notifications
+    places: Favorite places
+    terms: Terms
+  FavoritePlaceRow:
+    addAnotherPlace: Add another place
+    # deleteThisPlace and editThisPlace are aria/tooltip texts.
+    deleteThisPlace: Delete this place
+    editThisPlace: Edit this place
+    setAddressForPlaceType: Set your {placeType} address
+  FavoritePlaceScreen:
+    addNewPlace: Add new place
+    editPlace: Edit {placeName}
+    editPlaceGeneric: Edit place
+    invalidAddress: Please set a location for this place.
+    invalidName: Please enter a name for this place.
+    nameAlreadyUsed: You are already using this name for another place. Please enter a different name.
+    placeNotFound: Place not found
+    placeNotFoundDescription: Sorry, the requested place was not found.
+  FavoritePlacesList:
+    description: Add the places you frequent often to save time planning trips:
+  FormNavigationButtons:
+    ariaLabel: Form navigation
+  ItinerarySummary:
+    fareCost: "{useMaxFare, select, 
+      true {{minTotalFare} - {maxTotalFare}}
+      other {{minTotalFare}}
+    }"
+  NarrativeItinerariesHeader:
+    numIssues: "{issueNum, number} {issueNum, plural, 
+        one {issue}
+        other {issues}
+    }"
+    # Note to translator: resultText is width-constrained
+    # (about half pane width)
+    resultText: "{pending, select,
+      true {Finding your options...}
+      other {
+        {itineraryNum, number} {itineraryNum, plural,
+          one {itinerary found}
+          other {itineraries found}
+        }
+      }
+    }"
+    selectArrivalTime: Arrival time
+    selectBest: Best option
+    selectCost: Cost
+    selectDepartureTime: Departure time
+    selectDuration: Duration
+    selectWalkTime: Walk time
+    titleText: "{pending, select, 
+      true {Finding your options...}
+      other {
```

Comment:
Tricky: the new line inside the formatting string will be included in the text
![image](https://user-images.githubusercontent.com/56846598/128376843-797f497a-0650-4e48-ba8a-df466bf24e1e.png)

Reply:
Wow that is tricky. Updated that in https://github.com/opentripplanner/otp-react-redux/pull/408/commits/f8572ec523f76e4d937d1d53f088147f0326735d